src/status_im/ui2/screens/chat/home.cljs

```diff
@@ -221,81 +316,84 @@
+                                                  (if config/new-activity-center-enabled?
+                                                    (re-frame/dispatch [:navigate-to :activity-center])
+                                                    (re-frame/dispatch [:navigate-to :notifications-center])))}
+      [icons/icon :main-icons/notification2 {:color (colors/theme-colors colors/neutral-100 colors/white)}]]
      (when (pos? notif-count)
-       [rn/view {:style (merge (styles/counter-public-container) {:top 5 :right 5})
+       [rn/view {:style          (merge (styles/counter-public-container) {:top 5 :right 5})
                  :pointer-events :none}
         [rn/view {:style               styles/counter-public
                   :accessibility-label :notifications-unread-badge}]])]))
 
 (defn qr-button []
-  [quo2.button/button {:type :grey
+  [quo2.button/button {:type                :grey
                        :accessibility-label "qr-button"
-                       :size 32
-                       :width 32
-                       :style {:margin-left 12}
-                       :on-press #(do
-                                    (re-frame/dispatch [::qr-scanner/scan-code
-                                                        {:handler ::qr-scanner/on-scan-success}]))}
-   [icons/icon :main-icons/qr2 {:color (quo2.colors/theme-colors quo2.colors/neutral-100 quo2.colors/white)}]])
+                       :size                32
+                       :width               32
+                       :style               {:margin-left 12}
+                       :on-press            #(do
+                                               (re-frame/dispatch [::qr-scanner/scan-code
+                                                                   {:handler ::qr-scanner/on-scan-success}]))}
+   [icons/icon :main-icons/qr2 {:color (colors/theme-colors colors/neutral-100 colors/white)}]])
 
 (defn scan-button []
-  [quo2.button/button {:type :grey
-                       :size 32
-                       :width 32
+  [quo2.button/button {:type                :grey
+                       :size                32
+                       :width               32
                        :accessibility-label "scan-button"
-                       :on-press #(do
-                                    (re-frame/dispatch [::qr-scanner/scan-code
-                                                        {:handler ::qr-scanner/on-scan-success}]))}
-   [icons/icon :main-icons/scan2 {:color (quo2.colors/theme-colors quo2.colors/neutral-100 quo2.colors/white)}]])
+                       :on-press            #(do
+                                               (re-frame/dispatch [::qr-scanner/scan-code
+                                                                   {:handler ::qr-scanner/on-scan-success}]))}
+   [icons/icon :main-icons/scan2 {:color (colors/theme-colors colors/neutral-100 colors/white)}]])
 
 (views/defview profile-button []
```

Comment:
[Nitpick] It'd be nice to get rid of these `views/defview` and `views/letsubs` macros while you're lightly touching their functions in new UI2 namespaces. I know they were copied from the old UI though.

Reply:
These parts of the UI needs update anyways, so gonna be fixed in other issues.